shell/common/rasterizer.cc

```diff
@@ -145,6 +145,10 @@ std::shared_ptr<flutter::TextureRegistry> Rasterizer::GetTextureRegistry() {
   return compositor_context_->texture_registry();
 }
 
+GrDirectContext* Rasterizer::GetGrContext() {
+  return surface_->GetContext();
```

Comment:
```suggestion
  return surface_ ? surface_->GetContext() : nullptr;
```

Reply:
nullsafety for c++ when?

Reply:
Something something billion dollar mistake